docs/methoddocs/Detectionlists.md

```diff
@@ -0,0 +1,25 @@
+# Detection Lists
```

Comment:
None of the other files in `methoddocs` begin with a capital letter.
This should be `detectionlists.md`.

Reply:
Otherwise, this looks good!

Reply:
Yes, I have fixed this but as mentioned in other comment, I am facing an error and hence have not pushed the change.